PrReviewCode/Program.cs

```diff
@@ -4,6 +4,8 @@ public class Program
 {
     public static void Main(string[] args)
     {
+        throw new Exception("Test Exception");
```

Comment:
Unconditional exception throw here will crash execution. Ensure this is intentional or remove the exception.

 **SUGGESTION:**

```suggestion
 // remove or conditionally guard this exception
```